### DISCUSSION TITLE
/software-systems-engineering/conways-law-and-consequences/

### DISCUSSION THREAD

**Discussion by giscus** [thenewobjective/thenewobjective.github.io#157] [Announcements]
# Conway's Law and Consequences

The personal website of Michael L Haufe

https://thenewobjective.com/software-systems-engineering/conways-law-and-consequences

**Comment by mlhaufe** [comment]
This article has been shared on LinkedIn:

https://www.linkedin.com/pulse/conways-law-consequences-michael-haufe-oltre/
